git_machete/client.py

```diff
@@ -1855,68 +1855,79 @@ def check_that_forkpoint_is_ancestor_or_equal_to_tip_of_branch(
                 f"Forkpoint {forkpoint_sha} is not ancestor of or the tip "
                 f"of the {branch} branch.")
 
-    def checkout_github_prs(self, pr_no: int) -> None:
+    def checkout_github_prs(self, pr_no: Optional[List[int]], *, all_opened_prs: bool = False, my_opened_prs: bool = False, opened_by: str = None) -> None:
```

Comment:
TBD: default argument values or not?

Because check if there is only one option passed is placed in argument parser, hence there is no such a check when use `checkout_github_prs` from  API.